### ISSUE TITLE
[BUG] Combined Widget Stack displays "empty" widgets

### ISSUE THREAD

**Issue by berkhornet** [jurialmunkey/skin.arctic.fuse#116]
### Skin section

Home Widgets

### Current Behavior

When using the "Combine Widget Stack" option, if there is nothing for the widget to display (for example - In-Progress Movies") and empty widget is displayed with the selector list in a different position. Normally in this the case the widget is simply not displayed.

### Expected Behavior

Do not display an "empty" widget on the widget selector list.

### Steps To Reproduce

1, Set the "Combiner Widget Stack" option.
2, Define a widget that will return no results

### Screenshots and Additional Info

![screenshot00003](https://github.com/jurialmunkey/skin.arctic.fuse/assets/67786207/f46c3b73-8cac-4483-a727-a9c339493f6a)

### Checklist

- [X] I have made a separate issue for each bug that I am reporting.
- [X] I have searched the issues and this bug has not been reported.
- [X] I am using the most recent skin version.
- [X] I have confirmed that this bug does not occur in the default skin.

**Comment by jurialmunkey** [comment]
I knew about this one before I pushed it but so far haven't found a good solution.

The problem is that all containers start with 0 items, so hiding the button when the widget has 0 items means no widgets will ever be shown. And there's no real way to check the number of items only after the widget loads.

At the moment, the best I've managed is having the widget display a blank "None" item. That'll be in next update and it is definitely better than it jumping up like that, but still not ideal.

**Comment by berkhornet** [comment]
@jurialmunkey - many thanks for your reply. Your solution seems a good compromise for now!

**Comment by jurialmunkey** [comment]
For the most part, this will be addressed in v0.4 which is currently available on the Omega Testing branch.
https://github.com/jurialmunkey/skin.arctic.fuse/archive/refs/heads/omega_testing.zip
 
v0.4 no longer uses the SkinShortcuts addon and the new script is not backwards compatible, so it won't inherit old setup. I'm working on an import addon to assist in the migration, so once that's ready I'll be pushing it to my Kodi repo -- but if you want to install early you can get from the branch (it won't touch your old config so you can still roll back if needed).

**Comment by berkhornet** [comment]
many thanks. Will give this a try in a couple of days as I am travelling now.
________________________________

**Comment by jurialmunkey** [comment]
v0.4 now in main repo.
